Preparing Chamada Aberta para Propostas (edição em português)

anisa-hawes opened the pull request:
I'm creating a draft blog post for our forthcoming call for proposals (edição em português)

Closes #3590 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [ ] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

anisa-hawes committed:
Create 2025-06-18-chamada-aberta-para-propostas.md

Create draft blog post.

Commit:
Merge branch 'gh-pages' into call-for-proposals-portugues

anisa-hawes committed:
Update 2025-06-18-chamada-aberta-para-propostas.md

Correct line break, lines 13-14

ericbrasiln commented:
Querida @programminghistorian/portuguese-team 

Subimos a primeira versão do texto da chamada aberta para novas lições e traduções da edição em português do *Programming Historian*. O rascunho completo está disponível nesse PR #3591 

Nosso cronograma prevê:

* Revisão do texto da chamada até **31 de julho**
* Lançamento da chamada no site por volta de **04 de agosto**
* Período de recebimento de propostas até **31 de outubro**

Gostaria de contar com a colaboração de vocês para revisar e aprimorar o texto nesses próximos dias. Um ponto em que especialmente precisamos de ajuda é na **ampliação da lista de temas e abordagens metodológicas** que gostaríamos de incentivar — essa seção pode ser fortalecida com base nos debates e lacunas em nossa lições e na comunidade lusófona.

Podem deixar comentários diretamente aqui.

Desde já, agradeço demais a atenção e a parceria!

Abraços

DanielAlvesLABDH commented on _posts/2025-06-18-chamada-aberta-para-propostas.md:
```diff
@@ -0,0 +1,72 @@
+
+A edição em português do *Programming Historian* busca propostas que preencham lacunas em nosso diretório de lições. Explore [nosso acervo](/pt/licoes/) para descobrir o que já está disponível e pensar no que você pode acrescentar.
+
+Se o método ou abordagem que você gostaria de abordar já está contemplado nas edições em [espanhol](/es/lecciones/), [francês](/fr/lecons/) ou [inglês](/en/lessons/) do *Programming Historian*, também aceitamos propostas para traduzi-las para o português. Consulte o nosso [Mapa de Traduções](/translation-concordance) para identificar opções de tradução. Nesta chamada, **propostas de \[ORIGINAIS/T ...
```
"gostaria de desenvolver" em vez de "gostaria de abordar" para evitar a repetição de "abordagem" e "abordar"

anisa-hawes replied:
Obrigada, @DanielAlvesLABDH. I've made this adjustment.

cc. @ericbrasiln.

DanielAlvesLABDH commented on _posts/2025-06-18-chamada-aberta-para-propostas.md:
```diff
@@ -0,0 +1,72 @@
+* Aplicação de IA generativa e desenvolvimento de RAG (*Retrieval-Augmented Generation*) para a pesquisa com fontes históricas
+* Aplicação de técnicas de Processamento de Linguagem Natural (PLN/NLP) em conjuntos de fontes históricas
+* Construção e uso de grafos de conhecimento (*knowledge graphs*) para pesquisas em humanidades
+* [ABERTO A OUTRAS SUGESTÕES DE TEMAS OU ABORDAGENS A PARTIR DAS CONSTRIBUIÇÕES DA COMUNIDADE]
```
Aqui na NOVA FCSH teríamos interesse em ter mais lições disponíveis sobre análise espacial, ferramentas de mapeamento ou de extração de entidades geográficas de textos, por exemplo

anisa-hawes replied:
Olá @ericbrasiln. I've added in Daniel's suggested theme as:

- _Uso de análise espacial, ferramentas de mapeamento ou de extração de entidades geográficas de textos_ (to adjust/phrase differently as you wish, of course) ☺️

DanielAlvesLABDH reviewed (commented):
Olá Eric! Acho que o texto está muito bom. Deixei apenas dois breves comentários, meras sugestões, claro. Um abraço

anisa-hawes committed:
Update 2025-06-18-chamada-aberta-para-propostas.md

Adjust wording, line 32
Add theme, line 39

anisa-hawes committed:
Update formulario.proposta.licao.txt

Update plain text form.

anisa-hawes committed:
Update 2025-06-18-chamada-aberta-para-propostas.md

Add dates, lines 13-4
Update links, lines 51 and 73

ericbrasiln committed:
Expanded list of suggested topics in CFP (Portuguese edition)

Added four new thematic areas to highlight regional and decolonial approaches:
- computational methods for paleography and manuscript analysis,
- data preservation and modeling for community and Indigenous/Maroon archives,
- digital workflows in low-infrastructure contexts,
- digital humanities and decolonial pedagogies.

These additions aim to diversify the call and reflect research practices from Latin America and Lusophone Africa.

ericbrasiln commented:
dear @anisa-hawes and @DanielAlvesLABDH 

I’ve just expanded the list of suggested topics in the latest commit, with a focus on including approaches rooted in Latin American and African contexts.

anisa-hawes committed:
Update 2025-08-06-chamada-aberta-para-propostas.md

- Update launch date
- Update blog post date

anisa-hawes committed:
Update 2025-08-06-chamada-aberta-para-propostas.md

- Update date, line 52
- Update link, line 76

Commit:
Merge branch 'gh-pages' into call-for-proposals-portugues

anisa-hawes commented:
Olá @ericbrasiln,

Following our conversation about how we could adapt question 9 of the proposal form, I have prepared a draft. 

What I am thinking is that we could keep the question as it is, and adjust the clarification statement following question so that it reads:

> Se você está propondo uma lição original, como o método ou ferramenta poderia ser aplicado ou adaptado a dados em outras línguas além do português?
> 
> Celebramos metodologias e ferramentas que possam ser utilizadas em contextos de pesquisa multilíngues, mas ainda assim estaríamos interessados em receber a sua proposta se a metodologia ou ferramenta que pretende ensinar funcionar particularmente bem (ou apenas) em português.

Would this be correct in Portuguese? Let me know what you think, and I'll update it across the Google Form and the plain text form. 

Thank you, 
Anisa

ericbrasiln commented:
That sounds great, @anisa-hawes.
I might adjust it to:

>Valorizamos metodologias e ferramentas aplicáveis em contextos de pesquisa multilíngues; ainda assim, estamos interessados em receber sua proposta caso a metodologia ou ferramenta que você pretende ensinar funcione especialmente bem (ou mesmo apenas) em português.

anisa-hawes committed:
Update formulario.proposta.licao.txt

Adjust clarifying note, Q9 (line 58).

Commit:
Merge branch 'gh-pages' into call-for-proposals-portugues

Commit:
Merge branch 'gh-pages' into call-for-proposals-portugues
